### ISSUE TITLE
Documentation 

### ISSUE THREAD

**Issue by benasher44** [circa10a/easy-soap-request#63]
Is there documentation on how to use this? Is this really just a call to axios with no other logic?

**Comment by circa10a** [comment]
Hi @benasher44 ! The usage documentation can be found in the README.md of the project.

To answer your question. yes, it's really just an axios wrapper.

Years ago when I was working at a large company with lots of legacy tech, I found myself trying to figure out how to consume lots of different SOAP API's with more modern tech (not java). Documentation was fairly sparse on the internet as well as internal to the company so I spent more time than I'd like to admit trying to figure out how to use SOAP. Once I figured out it's just a simple POST request, I wrote [this article](https://betterprogramming.pub/how-to-perform-soap-requests-with-node-js-4a9627070eb6) to hopefully make figuring out SOAP API's for easier for the newbie's that followed me. This library has become decently popular over time so I think it's served its purpose in helping others figure out SOAP. Whether that's by usage of the library or just the example code in the wrapper method.
